controllers/defaults/placements.go

```diff
@@ -76,7 +76,7 @@ var (
 			},
 			PodAntiAffinity: &corev1.PodAntiAffinity{
 				PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
-					getWeightedPodAffinityTerm(100, "rook-ceph-mds"),
+					getMdsWeightedPodAffinityTerm(100, "ocs-storagecluster-cephfilesystem"),
```

Comment:
We need to pass the name of the actual filesystem instead of the hard coded value, right?

Reply:
this one is only for the default case

Reply:
UI/CLI needs to create a CR with there filesystem value

Reply:
IIRC the name of the cephfilesystem CR is: `<storageCluster>-cephfilesystem`, and the name of the `storageCluster` could be different than the default if the CR is created directly by the user instead of the UI. If that's the case, we can't hard code the filesystem name here.

Reply:
correct updated